src/javascript/app/common/form_manager.js

```diff
@@ -3,13 +3,23 @@ const BinarySocket     = require('../base/socket');
 const getHashValue     = require('../../_common/url').getHashValue;
 const isEmptyObject    = require('../../_common/utility').isEmptyObject;
 const showLoadingImage = require('../../_common/utility').showLoadingImage;
+const getElementById   = require('../../_common/common_functions').getElementById;
 
 const FormManager = (() => {
     const forms = {};
 
-    const initForm = (form_selector, fields, needs_token) => {
+    const initForm = (form_selector, fields, needs_token, should_set_trading_password) => {
         const $form = $(`${form_selector}:visible`);
-        const $btn  = $form.find('button[type="submit"]');
+        let $btn;
+        if (form_selector === '#frm_trading_password') {
+            if (should_set_trading_password) {
+                $btn = $form.find(getElementById('set_trading_btn'));
```

Comment:
```suggestion
                $btn = $form.find('#set_trading_btn');
```
Here we have jQuery element so it seems no need to use `getElementById` because we'll not get an error if the element doesn’t exist.

Reply:
@amirghadimi-fs we won't get an error, because this is our built in function and not document.getelementbyid. but your point is valid that this file is using jquery and should stick to jquery.